src/public/modules/forms/services/submissions.client.factory.js

```diff
@@ -179,8 +181,13 @@ function SubmissionsFactory($q, $http, $timeout, $window, GTag, FormSgSdk) {
                 }
 
                 if (csvRecord.submissionData) {
-                  // accumulate dataset if it exists, since we may have status columns available
-                  experimentalCsvGenerator.addRecord(csvRecord.submissionData)
+                  try {
+                    // accumulate dataset if it exists, since we may have status columns available
+                    experimentalCsvGenerator.addRecord(csvRecord.submissionData)
```

Comment:
should we rename experimentalCsvGenerator now that it has already been in the application for a while